www/deprecation_policy.md

```diff
@@ -0,0 +1,44 @@
+---
+layout: contribute
+title: Apache Cordova Deprecation Policy
+---
+
+# Deprecation and Archiving Policy
+
+Apache Cordova has been around quite some time, having been started back in August 2008. As it consists of many individual "parts", some of those stopped being useful for any number of reasons. This "Deprecation and Archiving Policy" describes why, when and how projects can be deprecated and archived - and what that actually means.
+
+## Deprecation Policy
+
+When a piece of Apache Cordova stops being useful or necessary we vote on deprecating it on our [dev mailing list](/contact/).
+
+### Reasons for Deprecation
```

Comment:
Any other reasons for Deprecation that were applied in the past?

Reply:
I remember "we" deprecated & dropped support for Amazon since the Amazon folks had dropped out, told people to just do a special Android build.

Reply:
That's what I thought of for "A platform was deprecated itself by its producer" - or what do mean by "dropped out"?

Reply:
I guess I was not clear enough:
- By "Amazon folks" I meant the people who made the `cordova-amazon` platform repo and the people who supported the `cordova-amazon` platform.
- By "dropped out" I meant "quit updating and supporting `cordova-amazon`". This was my understanding from several years ago, way before I was a member. (I was not part of "we" back then.)

For reference:
- <https://github.com/apache/cordova-discuss/issues/32>
- <https://groups.google.com/forum/#!topic/phonegap/6oZfxBc4hPc>

Reply:
Exactly, Amazon itself had "deprecated" `fireos` (and switched to more "stock" Android) which meant Cordova could also remove the platform. At least that's how I remember it. 

If you have a sentence I could add to the bullet list, I would be fine with adding it of course.